build from source error: undefined: iopodman.ContainerStats

/kind bug

**Description**

I'm running into an error when building `podman-remote` from source.

**Steps to reproduce the issue:**

```
git clone --branch v1.8.0 https://github.com/containers/libpod $GOPATH/src/github.com/containers/libpod && cd $GOPATH/src/github.com/containers/libpod && LDFLAGS="-s -w" make varlink_generate podman-remote BUILDTAGS="selinux seccomp apparmor"
```

**Describe the results you received:**

```
Cloning into '/go/src/github.com/containers/libpod'...
touch .gopathok
GO111MODULE=on go build -mod=vendor  -gcflags 'all=-trimpath=/go/src/github.com/containers/libpod' -asmflags 'all=-trimpath=/go/src/github.com/containers/libpod' -ldflags '-X github.com/containers/libpod/libpod/define.gitCommit=2ced9094d4728dd09f60a177faa32339a8d0f721 -X github.com/containers/libpod/libpod/define.buildInfo=1581088958 -X github.com/containers/libpod/libpod/config._installPrefix=/usr/local -X github.com/containers/libpod/libpod/config._etcDir=/etc -extldflags "-s -w"' -tags "selinux seccomp apparmor remoteclient" -o bin/podman-remote github.com/containers/libpod/cmd/podman
# github.com/containers/libpod/pkg/varlinkapi
pkg/varlinkapi/remote_client.go:12:49: undefined: iopodman.ContainerStats
# github.com/containers/libpod/cmd/podman/shared
cmd/podman/shared/intermediate_varlink.go:70:42: undefined: iopodman.Create
cmd/podman/shared/intermediate_varlink.go:71:7: undefined: iopodman.Create
cmd/podman/shared/intermediate_varlink.go:317:34: undefined: iopodman.Create
make: *** [Makefile:191: bin/podman-remote] Error 2
```

**Describe the results you expected:**

Successful build, similar to `v1.7.0`?

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
Building... `v1.8.0`
```

**Output of `podman info --debug`:**

N/A

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

N/A

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Build container image: `golang:1.13.6-alpine3.11`

https://circleci.com/gh/marshallford/podman/39?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

--- a/Makefile
+++ b/Makefile
@@ -582,7 +582,7 @@ install.libseccomp.sudo:
 
 
 cmd/podman/varlink/iopodman.go: .gopathok cmd/podman/varlink/io.podman.varlink
-ifeq ("$(shell uname -o)", "GNU/Linux")
+ifneq (,$(findstring Linux,$(shell uname -o)))
 	# Only generate the varlink code on Linux (see issue #4814).
 	GO111MODULE=off $(GO) generate ./cmd/podman/varlink/...
 endif

